hapi/hedera-protobufs/services/basic_types.proto

```diff
@@ -1837,3 +1837,47 @@ message ConsensusTokenFeeScheduleAllowance {
      */
     uint64 amount_per_message = 5;
 }
+
+/**
+ * A unique, composite, identifier for a topic allowance.
+ *
+ * Each topic allowance SHALL be uniquely identified by a TopicAllowanceId.
+ * A TopicAllowanceId SHALL be recorded when created and MUST be provided in any transaction
+ * that would modify that topic allowance (such as a `consensusApproveAllowance`).
+ */
+message TopicAllowanceId {
+    /**
+     * The token that the allowance pertains to.
```

Comment:
This does not match [style guidelines](https://github.com/hashgraph/hedera-protobufs/blob/hip-1037-documentation/documents/Specification-Format-Style-Guidelines.md) and has insufficient detail.
This comment applies equally to every field added to this file for this PR.